CHANGES.md

```diff
@@ -1,5 +1,88 @@
+ - Fix multiple WSL issues related to installation and display.  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3573)
+ - Check CPU attributes to determine whether to install an AVX-less version of TensorFlow (M1 Macs, older Linux CPUs).  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3569)
+ - Remove `futures` from our `requirements.txt` file.  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3548)
+ - Use Python 3.7 instead of Python 3.6 for SCT's conda environment.  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3361)
+
+**DOCUMENTATION**
+
+ - Add studies citing SCT.  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3656)
+ - Replace `sct_maths` with `sct_separate_dmri_separate_dwi_and_b0` in tutorial dMRI preprocessing.  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3643)
+ - Help differenciate Linux and WSL user on install documentation.  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3641)
+ - Add Uhrenholt et al..  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3639)
+ - Use furo-theme sanctioned dark mode colour controls..  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3632)
+ - Restore "Edit on GitHub" button.  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3624)
+ - Replace SCT Course 2020 iCloud link with Google Drive link.  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3623)
+ - **sct_deepseg_sc:** `sct_deepseg_sc`: Indicate default value for `-brain`.  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3617)
+ - **sct_register_to_template:** Update help for 2+ labels usage, and change link from iCloud to RTD.  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3603)
+ - Fix typo in `studies.rst` (Kinawy -> Kinany).  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3589)
+ - Remove misleading `FSLeyes` installation instructions.  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3577)
+ - `README.rst`: Fix typo.  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3576)
+ - `windows.rst`: Recommend echoing DISPLAY to ~/.profile, not .profile.  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3563)
+ - Remove leftover unnecessary `R|` formatting.  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3561)
+ - Add Hernandez et al..  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3557)
+ - Add Staud et al..  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3551)
```

Comment:
All of these references which end in "et al." introduce redundant periods, but I don't know if we care